src/main/resources/benchmarks/tpch/gather_schema_stats_oracle.sql

```diff
@@ -0,0 +1,2 @@
+-- Copyright (c) 2023, Oracle and/or its affiliates.
+BEGIN dbms_stats.gather_schema_stats(ownname => 'benchbase', estimate_percent => dbms_stats.auto_sample_size, method_opt => 'for all columns size AUTO'); END;;
```

Comment:
Looks like this file is repeated.  Possible to consolidate to a single copy?

Reply:
Consolidated to `src/main/resources/benchmarks/gather_schema_stats_oracle.sql` now.